components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -20,6 +22,22 @@
 CLP_METADATA_TABLE_PREFIX = 'clp_'
 SEARCH_JOBS_TABLE_NAME = 'distributed_search_jobs'
 
+# Package modes
+class StorageEngine(KebabCaseStrEnum):
+    CLP = auto()
```

Comment:
Sorry, didn't think of this before, but technically, our rules for capitalizing acronyms dictate that this should be `Clp`. I think in anticipation of the CLP-S storage engine, this also helps since it would be `ClpS` rather than `CLPS`.

Reply:
Actually, scratch that. Our rules for Python are different than C++. It would be `CLP_S`.

Reply:
Yeah, and I verified that this KebabCaseStrEnum turns CLP_S into clp-s which is what we want I think

Reply:
Neat